debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java

```diff
@@ -68,13 +68,23 @@
             .withDefault(DEFAULT_POLL_INTERVAL_MILLIS)
             .withValidation(Field::isPositiveInteger);
 
+    public static final Field SNAPSHOT_DELAY_MS = Field.create("snapshot.delay.ms")
+        .withDisplayName("Snapshot Delay (milliseconds)")
+        .withType(Type.LONG)
+        .withWidth(Width.MEDIUM)
+        .withImportance(Importance.LOW)
+        .withDescription("The number of milliseconds to delay before a snapshot will begin.")
+        .withDefault(0L)
+        .withValidation(Field::isNonNegativeLong);
+
     private final Configuration config;
     private final boolean emitTombstoneOnDelete;
     private final int maxQueueSize;
     private final int maxBatchSize;
     private final Duration pollInterval;
     private final String logicalName;
     private final String heartbeatTopicsPrefix;
+    private final Duration snapshotDelayMs;
```

Comment:
I'd just name it `snapshotDelay`, as `Duration` isn't bound to any unit.

Reply:
done